IntegrationTests/tests_02_syscall_wrappers/test_02_unacceptable_errnos.sh

```diff
@@ -47,10 +47,12 @@ if pipeErr != 0 {
 }
 let makeEBADFHappen =  CommandLine.arguments.dropFirst().first == .some("EBADF")
 let makeEFAULTHappen = CommandLine.arguments.dropFirst().first == .some("EFAULT")
-var whatevs: UInt8 = 123
+let makeEINVALHappen = CommandLine.arguments.dropFirst().first == .some("EINVAL")
+var whatevs: UInt8 = makeEINVALHappen ? -1 : 123
```

Comment:
This still won't work. `EINVAL` is triggered by the _pointer_ being negative: this is the `pointee`, and so unaffected. Also, this still won't compile. :smile: